repose-aggregator/src/docs/asciidoc/architecture/system-model.adoc

```diff
@@ -170,6 +170,44 @@ If set to localhost, any physical node will match and run the *Repose* node.
      In this case, no services are being used.
 <13> Defines the required one default endpoint.
 
+==== Streaming Request Body to the Origin Service
+Streaming a request body to the origin service is accomplished by using the `chunked` transfer encoding.
+
+[source,xml]
+.partial http-connection-pool.cfg
+----
+<system-model xmlns="http://docs.openrepose.org/repose/system-model/v2.0">
+  <repose-cluster id="repose_cluster1"
+                  chunked-encoding="auto"> <!--1-->
```

Comment:
So if we do move this to the system model, it makes more sense for it to be on the destinations, not the cluster. It's possible that they may have some destinations that don't support chunking, and if i recall the value is really only for communication to the origin service.

Reply:
I thought that too, actually. I originally put it on `destinations` since there is not currently a way to disable chunked encoding for one `destination` but enable it for another. Then I moved it here since you can only have one set of destinations anyway, and this attribute living next to the `rewrite-host-header` attribute made sense to me at the time.

That said, I'm totally cool with dropping this onto `destinations` again. I would actually be happy to drop it onto `destination`, but that would be an enhancement that requires a bit more work.

Reply:
yeah, i was suggesting it should be on the individual destinations.

Reply:
Sure. How do we want to handle per-destination chunking, though?

As of this PR, it would be non-trivial to implement. However, once we drop the routing complications added by supporting the WAR deployment, it becomes much more trivial. Ideally, we would add per-destination chunking as a feature after both of those stories, but I realize I am already messing with the schema here, so...